MipFingerprint.nf

```diff
@@ -113,6 +116,7 @@ process VersionLog {
     tag {"VersionLog"}
```

Comment:
is there a reason that the parameter ${analysis_id}"} is missing in only this pull request (MIP, compared to the WGS and WES pull requests of the same function)

Reply:
No particular reason, probably changed in WES / WGS after implementing it in this workflow. Added it for consistency.